Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -532,21 +532,21 @@ granularity.info.legend=Your choice\:
-granularity.info.textbox.weeks=Each process represents one week of the course of publication of the newspaper and contains all issues actually published this week. A week starts with Monday. When New Year's Day does not fall onto a Monday, two processes for the two week parts in the respective calendar years are generated. Since the DFG Viewer\u2019s calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the week, regardless of the day chosen, therefore\u2212in many cases\u2212on the wrong day. Note also that week boundaries regularly do not correspond with monthly limits.
-granularity.info.textbox.years=Each process represents a single calendar year of the appearance curve of the newspaper that contains all this year actually published editions of the newspaper. Since the DFG Viewer\u2019s calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the year, regardless of the day chosen, therefore\u2212in the very most cases\u2212on the wrong day.
+granularity.info.textbox.quarters=Each process represents a single quarter of a year of the appearance curve of the newspaper that contains all this quarter actually published editions of the newspaper. The quarters start with the 1st of January, April, July and October, respectively. Since the DFG Viewer's calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the quarter, regardless of the day chosen, therefore−in the very most cases−on the wrong day.
+granularity.info.textbox.weeks=Each process represents one week of the course of publication of the newspaper and contains all issues actually published this week. A week starts with Monday. When New Year's Day does not fall onto a Monday, two processes for the two week parts in the respective calendar years are generated. Since the DFG Viewer's calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the week, regardless of the day chosen, therefore−in many cases−on the wrong day. Note also that week boundaries regularly do not correspond with monthly limits.
+granularity.info.textbox.years=Each process represents a single calendar year of the appearance curve of the newspaper that contains all this year actually published editions of the newspaper. Since the DFG Viewer's calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the year, regardless of the day chosen, therefore−in the very most cases−on the wrong day.
```

Comment:
```suggestion
granularity.info.textbox.years=Each process represents a single calendar year of the appearance curve of the newspaper that contains all this year actually published editions of the newspaper. Since the DFG Viewer's calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the year, regardless of the day chosen, therefore - in the very most cases - on the wrong day.
```